### PR TITLE
Improved unit tests

### PR DESCRIPTION
**Pull request by enahum**
@hmhealey this should do what we discussed with the unit tests, they are way easy to write and they get executed way faster too.

### CODE REVIEW

**Comment by enahum** [comment]
@thomchop if you have the time can you review this PR?

File: test/service/actions/channels.test.js [hunk 1]
```diff
@@ -4,508 +4,375 @@
...
-                display_name: 'Redux Test',
-                purpose: 'This is to test redux',
-                header: 'MM with Redux',
-                type: 'O'
-            };
-
-            Actions.createChannel(channel, TestHelper.basicUser.id)(store.dispatch, store.getState);
-        });
+    let store;
+    let secondChannel;
```
Comment (hmhealey) [inline test/service/actions/channels.test.js]: Should this be out here? I think I only saw it in one test.

Comment (enahum) [inline test/service/actions/channels.test.js]: yes, its to be able to re-use the second channel id

Comment (hmhealey) [inline test/service/actions/channels.test.js]: Ah okay

**Pull request merged**